TestOutput/EventHeaderInterceptorLE64.dat.linux.json

```diff
@@ -259,4 +259,118 @@
   { "n": "TestProviderCpp:Value:char32_t*", "0": "", "hello": "hello", "meta": { "level": 5 } },
   { "n": "TestProviderCpp:Value:cchar32_t*", "0": "", "hello": "hello", "meta": { "level": 5 } },
   { "n": "TestProviderCpp:Value:wchar_t*", "0": "", "hello": "hello", "meta": { "level": 5 } },
-  { "n": "TestProviderCpp:Value:cwchar_t*", "0": "", "hello": "hello", "meta": { "level": 5 } } ]
+  { "n": "TestProviderCpp:Value:cwchar_t*", "0": "", "hello": "hello", "meta": { "level": 5 } },
```

Comment:
Note: Added new test cases that use `EventHeaderDynamic` to output blobs of various lengths using the `binary` and `string` encodings, once for each format.